instrumentation/ruby_kafka/lib/opentelemetry/instrumentation/ruby_kafka/patches/client.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module RubyKafka
+      module Patches
+        # The Client module contains the instrumentation patch the Client#deliver_message and Client#each_message methods.
+        module Client
+          def deliver_message(value, key: nil, headers: {}, topic:, partition: nil, partition_key: nil, retries: 1)
+            attributes = {
+              'messaging.system' => 'kafka',
+              'messaging.destination' => topic,
+              'messaging.destination_kind' => 'topic'
+            }
+
+            attributes['messaging.kafka.message_key'] = key if key
+            attributes['messaging.kafka.partition'] = partition if partition
+
+            tracer.in_span('send', attributes: attributes, kind: :producer) do
```

Comment:
Looking at the spec, it appears the span name should be of the form `<destination name> <operation name>`.  In the case of this span, it should be `"#{topic} send"`.

See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md#span-name